docs/source/changelog.rst

```diff
@@ -34,6 +34,7 @@ Changelog
         * Refactor get_pandas_data_slice to take single entity (:pr:`547`)
         * Updates TimeSincePrevious and Diff Primitives (:pr:`561`)
         * Remove unecessary time_last variable (:pr:`546`)
+        * Remove max_hlevel from DeepFeatureSynthesis (:pr:`608`)
```

Comment:
Since this was merged into this PR it doesn't need to be in the changelog at all, right?

Reply:
We're not strict about it. This more about communicating what change rather than a one to one match to what merged in master. Perhaps add the PR as an additional PR number under changelog line for this PR.